cartographer/mapping/3d/submap_3d.cc

```diff
@@ -272,6 +297,15 @@ void Submap3D::InsertRangeData(const sensor::RangeData& range_data,
   range_data_inserter.Insert(transformed_range_data,
                              low_resolution_hybrid_grid_.get());
   set_num_range_data(num_range_data() + 1);
+  if (rotational_scan_matcher_histogram_.rows() == 0) {
+    rotational_scan_matcher_histogram_.resize(scan_histogram_in_gravity.size());
+    rotational_scan_matcher_histogram_.setZero();
+  }
+  const float yaw_to_submap_from_gravity =
```

Comment:
i think the name is a bit confusing, since it's the (gravity-aligned) yaw for the scan, relative to the submap's orientation, right? yaw_in_submap?